CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Fixes
 
+### Allow Kit specific patterns to be used with the unbranded template
```

Comment:
@hannalaakso if you get time could you check these steps for factual accuracy

I've assigned @m-green to review this when he has time.

Reply:
Looks good, thanks @nickcolley 👍 I'm not sure we need to mention step 3 as they'll get this change if they follow the kit update instructions.